dataframe/at_indexer.go

```diff
@@ -0,0 +1,97 @@
+package dataframe
+
+import (
+	"fmt"
+
+	"go.starlark.net/starlark"
+)
+
+// AtIndexer is returned by dataframe.At and can index to any cell
+type AtIndexer struct {
+	owner *DataFrame
+}
+
+// compile-time interface assertions
+var (
+	_ starlark.Value     = (*AtIndexer)(nil)
+	_ starlark.Mapping   = (*AtIndexer)(nil)
+	_ starlark.HasSetKey = (*AtIndexer)(nil)
+)
+
+// NewAtIndexer returns a new AtIndexer with the text values and name
+func NewAtIndexer(owner *DataFrame) *AtIndexer {
+	return &AtIndexer{owner: owner}
+}
+
+// Freeze prevents the index from being mutated
+func (ai *AtIndexer) Freeze() {
+}
+
+// Hash cannot be used with AtIndexer
+func (ai *AtIndexer) Hash() (uint32, error) {
+	return 0, fmt.Errorf("unhashable: %s", ai.Type())
+}
+
+// String returns the index as a string
+func (ai *AtIndexer) String() string {
+	return fmt.Sprintf("AtIndexer(...)")
```

Comment:
are we going to expand this output? If so I'd love a `TODO(dustmop)`

Reply:
AtIndexer objects are not printable in any reasonable way (python's pandas just shows the class name and memory address). I'll remove the 3 dots so it just looks opaque.